content/developer/theming/about-php.md

```diff
@@ -1,91 +0,0 @@
----
```

Comment:
@Charrondev Don't delete the about.php page. If anybody has links to it, we don't want them to land on a 404 page. Mark it as deprecated, keeping relevant information and link to the new pages talking about the json format.

Reply:
I've kept the information in the add-on information page and added an alias to is. I've tested the redirect.

Reply:
Even better! Ask for a redirect from OPs

Reply:
The aliases field in the front-matter handles the redirects, so I've already put in redirects for any page I've moved or that are related to the old one.

Reply:
I've now made a separate PR for moving the smarty docs. It included redirects.

https://github.com/vanilla/docs/pull/146